spec/transaction/components/responses.yaml

```diff
@@ -18,6 +25,16 @@ components:
         application/json:
           schema:
             $ref: "./schemas.yaml#/components/schemas/assetWrite"
+    CollectionAsset:
+      description: >-
+        The response is a document consisting of one asset of the collection.
+      headers:
+        ETag:
+          $ref: "../../components/headers.yaml#/components/headers/ETag"
```

Comment:
For my understanding: Why does the `CollectionAssets` have no `headers` while `CollectionAsset` does?

Reply:
AFAIK the ETag header is only used if a single entity is returned, and does not really make sense if lists are returned.